tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -1568,6 +1569,11 @@ HRESULT DxcCompilerAdapter::WrapCompile(
       IFT(pArgs->AddArguments(EmbedDebugOpt, _countof(EmbedDebugOpt)));
     }
 
+#define SPIRV_TEST
+#ifdef SPIRV_TEST
```

Comment:
What if rather than a preprocessor definition, this was guarded by an environment variable (in addition to a command line switch IMO). That way, you could run existing command lines as-is, but still have it be modal that the code round-trips through SPIR-V.